src/univariate/discrete/negativebinomial.jl

```diff
@@ -82,6 +82,16 @@ kurtosis(d::NegativeBinomial{T}) where {T} = (p = succprob(d); T(6) / d.r + (p *
 
 mode(d::NegativeBinomial{T}) where {T} = (p = succprob(d); floor(Int,(one(T) - p) * (d.r - one(T)) / p))
 
+function kldivergence(p::NegativeBinomial, q::NegativeBinomial; kwargs...)
+    if p.r ≈ q.r
```

Comment:
Do we actually want approximate equality here? It seems a bit unclear how large the error of the result can become. Maybe safer:

```suggestion
    if p.r == q.r
```